Sources/ComposableArchitecture/Reducer/Reducers/PresentationReducer.swift

```diff
@@ -212,7 +212,7 @@ extension Reducer {
   ///     // ...
   ///   }
   ///
-  ///   var body: some Reducer<State, Action> {
+  ///   var body: some ReducerOf<Self> {
```

Comment:
Just for consistency with the one in `StackReducer.swift`

https://github.com/pointfreeco/swift-composable-architecture/blob/f0bae57f361b3611b48d7db0b78393e5c78be1fc/Sources/ComposableArchitecture/Reducer/Reducers/StackReducer.swift#L216